### DISCUSSION TITLE
I havent recieved my Student Devoloper package and benefits, when will I get them?

### DISCUSSION THREAD

**Discussion by rykersimpson** [community/community#116792] [GitHub Education]
### Select Topic Area

Question

### Body

I applied for the student developer pack, and I was approved on March 29th. It is currently April 4th, well over the specified 72 hours which was stated how long it would take. Is this a glitch? Does anybody know when I will receive the developer benefits?

**Comment by giovannivicentin** [comment]
Hello RykerSimpson,

Congratulations on getting approved for the GitHub Student Developer Pack! It's a fantastic resource with a lot of benefits, so it's understandable you're eager to get started.

It's unusual for there to be a delay beyond the specified 72 hours, but there are a few reasons this might happen, such as high demand, verification steps, or system issues. Here's what you can do to address the situation:

1. **Check Your Email (Including Spam/Junk):** Sometimes, the confirmation emails or follow-up instructions from GitHub or its partners can end up in the spam or junk folder. Make sure to check these folders for any missed communications.

2. **Verify Your Account Status:** Log back into your GitHub account and visit the Student Developer Pack section to see if there's any additional information or steps you might have missed. Sometimes, there might be additional verification steps or information required.

3. **GitHub Education Community:** While waiting for a response from GitHub support, you might want to search or ask in the [GitHub Education Community](https://education.github.community/). Other students or educators may have experienced similar delays and could have insights or solutions.

4. **GitHub Support:** If none of the above steps clarify the situation, reaching out to GitHub Support directly would be the best course of action. They can provide specific details about your account and the status of your Developer Pack access.

Given that it's just slightly over the expected timeframe, it could simply be a minor delay. However, it's definitely a good idea to start these checks now to ensure you can access your benefits as soon as possible.

Best of luck, and I hope you get access to your Developer Pack benefits very soon!

Best regards

**Comment by grandmas-favorite** [comment]
Hey @rykersimpson!  👋


Thanks for posting in the GitHub Community! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Packages category is a place for our community to discuss and provide feedback on the GitHub Packages tool, so you are more likely to get a useful response if you are posting your question(s) in the Education category. I'll move the discussion there!
